### DISCUSSION TITLE
Notification Text

### DISCUSSION THREAD

**Discussion by dvishal18** [HaylLtd/cordova-plugin-background-geolocation#179] [General]
I have Configured the notificationTitle and notificationText but it shows the default notification text and title

**Comment by HarelM** [comment]
So you probably misconfigured it.
Here's an example from my app:
https://github.com/IsraelHikingMap/Site/blob/de4eff0ad97189fa7ebafc5be9a36aefd93dd595/IsraelHiking.Web/src/application/services/geo-location.service.ts#L164

**Reply by dvishal18** [reply]
It was my SVN :D

**Comment by dvishal18** [comment]
Able to see the correct notification text
